### PR TITLE
Create textlint config

### PR DESCRIPTION
**Pull request by qjack001**
[Textlint](https://textlint.github.io) is a configurable linter for Markdown, plaintext, and HTML files. This PR sets up a preset following our own styleguide. If you like the idea, I will update the publish script and documentation to include it.

### CODE REVIEW

File: textlint/index.js [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+			allowLists: true,
+			allowWords: [
+				'bell',
+				'jit',
+				'jue',
+			],
+		},
+		'terminology': {
+			defaultTerms: false,
+			terms: [
```
Comment (qjack001) [inline textlint/index.js:40]: This is more or less just a copy-paste of the default terms provided by the extension. However, using the default terms removed the ability to edit or turn off certain cases. So a few removals, modifications, and additions to the default.

File: textlint/index.js [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+				'AVA',
+				'BrowserStack',
+				'Browsersync',
+				'Codecov',
+				'CodePen',
+				'CodeSandbox',
+				'DefinitelyTyped',
+				'EditorConfig',
+				'ESLint',
+				'Github',
```
Comment (qjack001) [inline textlint/index.js:54]: Purposely spelling "GitHub" as "Github"

File: textlint/index.js [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+				'GraphQL',
+				'GraphiQL',
+				'iOS',
+				'Javascript',
```
Comment (qjack001) [inline textlint/index.js:58]: Same here: "JavaScript" is technically correct, but "Javascript" just feels more right. It's our styleguide, we can do what we want!

File: textlint/index.js [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+				'PNG',
+				'SaaS',
+				'URL',
+				[ 'URL[\'’]?s', 'URLs' ],
+				[ 'an URL', 'a URL' ],
+				[ 'wi[- ]?fi', 'Wi-Fi' ],
+				'ID',
+				[ 'id[\'’]?s', 'IDs' ],
+				[ 'backwards compatible', 'backward compatible' ],
+				[ 'she or he', 'they' ],
```
Comment (qjack001) [inline textlint/index.js:115]: The regex-replacing here feels like the perfect tool to expand on these un-binary-ing rules, so a few more added (please look over and confirm).

File: textlint/index.js [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+				[ 'user[- ]name(s)?', 'username$1' ],
+				[ 'walk[- ]through', 'walkthrough' ],
+				[ 'white[- ]space', 'whitespace' ],
+				[ 'wild[- ]card(s)?', 'wildcard$1' ],
+				[ 'file-?type(s)?', 'file type$1' ],
+				[ 'micro-?frontend(s)?', 'micro frontend$1' ],
+				[ 'open-?source(ed?)', 'open source$1' ],
+				[ 'regexp?(s)?', 'regular expression$1' ],
+				[ 'style[- ]guide(s)?', 'styleguide$1' ],
+				[ 'Good[- ]byte', 'Goodbyte' ],
```
Comment (qjack001) [inline textlint/index.js:166]: :)

File: textlint/index.js [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+				[ 'touch[- ]screen(s)?', 'touchscreen$1' ],
+				[ 'user[- ]name(s)?', 'username$1' ],
+				[ 'walk[- ]through', 'walkthrough' ],
+				[ 'white[- ]space', 'whitespace' ],
+				[ 'wild[- ]card(s)?', 'wildcard$1' ],
+				[ 'file-?type(s)?', 'file type$1' ],
+				[ 'micro-?frontend(s)?', 'micro frontend$1' ],
+				[ 'open-?source(ed?)', 'open source$1' ],
+				[ 'regexp?(s)?', 'regular expression$1' ],
+				[ 'style[- ]guide(s)?', 'styleguide$1' ],
```
Comment (qjack001) [inline textlint/index.js:165]: As mentioned in this repo's readme: "styleguide" is wrong by all accounts except mine. It's its own thing, and it should get its own word.

File: textlint/index.js (hunk 4 above)
Comment (HoloPollock) [reply textlint/index.js:115]: Actually jack I need my gender neutral gliph. Jk jk Yah that seems good

Comment (HoloPollock) [reply textlint/index.js:115]: If you want you could add he or she and s(he)

Comment (HoloPollock) [reply textlint/index.js:115]: I see you did this already

**Review by HoloPollock — APPROVED** [review]

**Pull request merged**